docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,679 @@
+    "  This is the radius in cylinder coordinates.\n",
+    "- Define bin edges for the radius.\n",
+    "- Bin `table` by `z_pos` and the radius.\n",
+    "- Plot the result.\n",
+    "\n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4fa0b2a8-fd78-41d0-8c2e-27fda47c9e1b",
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "radius = sc.sqrt(x**2 + y**2)\n",
+    "table.coords['radius'] = radius\n",
+    "radius_edges = sc.linspace(\n",
+    "    'radius', radius.min().value, radius.max().value, num=13, unit='mm'\n",
+    ")\n",
+    "tmp = sc.bin(table, edges=[z_pos, radius_edges])\n",
+    "tmp.bins.sum().plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "922574fd-cc8e-407c-81d1-ce4cd3eeabc1",
+   "metadata": {},
+   "source": [
+    "## Multi-dimensional logical binning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0186673f-56f8-4daf-8cfc-e5de77ae4459",
+   "metadata": {},
+   "source": [
+    "Above we binned according to x, y, and z.\n",
+    "This reflects neither the physics nor the logical structure of the detectors and is generally not very useful.\n",
+    "The original table additionally contains information about the logical structure of the detector array.\n",
+    "In this case it is divided into modules, segments, counters, wires, and strips.\n",
+    "We define:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbc8d773-9271-46a6-8a39-eec7525f7463",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note that indices in the file are 1-based, not 0-based\n",
+    "groups = {\n",
+    "    dim: sc.arange(dim, 1, table.coords[dim].max().value + 1, unit=None, dtype='int64')\n",
+    "    for dim in ['module', 'segment', 'counter', 'wire', 'strip']\n",
+    "}\n",
+    "groups"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14a4ec01-4649-4dc3-ad2e-e92432e690e1",
+   "metadata": {},
+   "source": [
+    "Instead of using `sc.bin` with the `edges` keyword argument we can use the `groups` keyword argument to perform a binning based on discrete values.\n",
+    "The result is 5-D:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e800d4f1-5851-4394-af7c-bcf07d54e7d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.bin(table, groups=list(groups.values()))\n",
+    "binned"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dbcbc78-2f57-45b7-94b2-d8b42d8c0b0d",
+   "metadata": {},
+   "source": [
+    "### Exercise 3\n",
+    "\n",
+    "- Group `table` using `sc.bin` but only by strip and wire.\n",
+    "- Plot the result.\n",
+    "- The wire index increases with the cylinder radius, the strip index increases with z (or decreases with scattering angle).\n",
+    "  Explain the plot, e.g., why is intensity decreasing with increasing wire index?\n",
```

Comment:
This is nice for scattering people, but it might confuse people with a completely different background. And it is a physics question and not about binned data. So I think it should be removed.

Reply:
You mean the entire bullet point? Yes, I think I agree.

Reply:
Yes, the entire bullet.